HEN_HOUSE/scripts/configure_c++

```diff
@@ -165,7 +165,7 @@ if test $create_config=yes; then
 
 case $CXX in
 
-    *g++*) opt="-O2 -mtune=native";;# mingw32-g++ and g++4 also taken into account
```

Comment:
You'll also need to make this change in egs_configure/egs_tools.cpp to affect the GUI installer.

Reply:
Thank you Reid :)

Reply:
Tried to address using 7f67c92, wasn't sure about including corresponding flags for other compilers. Apparently recent versions `cl` don't have a `std=c++11` flag at all and the features are enabled by default: https://docs.microsoft.com/en-us/cpp/build/reference/std-specify-language-standard-version?view=vs-2019